src/kubelet/metric/resource_test.go

```diff
@@ -4,11 +4,19 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/newrelic/nri-kubernetes/src/definition"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/newrelic/nri-kubernetes/v2/src/definition"
+)
+
+const (
+	// Copied from old version of k8s.io/api/core/v1.
+	//
+	// NVIDIA GPU, in devices. Alpha, might change: although fractional and allowing values >1, only one whole device per node is assigned.
+	resourceNvidiaGPU v1.ResourceName = "alpha.kubernetes.io/nvidia-gpu"
```

Comment:
```suggestion
	resourceNvidiaGPUName v1.ResourceName = "alpha.kubernetes.io/nvidia-gpu"
```

Reply:
I'd suggest to add a `// TODO` here since we should revisit this later, to consider whether we should deprecate it or not.

Reply:
I get rid of that part, adding the deprecation note was quite confusing giving that we were not using that string at all